string id
string name
string previewURL
database_map_server/BasePoint[] points
database_map_server/BaseWall[] walls

